### ISSUE TITLE
Switch Expressions Enabled in Swift 5.8 Mode

### ISSUE THREAD

**Issue by jshier** [nicklockwood/SwiftFormat#1405]
Looks like the switch expressions feature is enabled under Swift 5.8 mode rather than 5.9 but if expressions are properly limited to 5.9. I also don't see an option to disable the switch version, just the rather limited `conditionalAssignment`. Seems like there should be an overall flag like `ifSwitchExpressions` (which is the proposal name for the feature).

More specifically, I'm getting switches in properties reformatted to the expression version:

```swift
var path: String {
    switch self {
    case .employee: return "employee"
    case .expenses: return "expenses"
    case .createExpense: return "expense"
    }
}
```

to 

```swift
var path: String {
    switch self {
    case .employee: "employee"
    case .expenses: "expenses"
    case .createExpense: "expense"
    }
}
```

**Comment by mitjacendak** [comment]
Bumping this, noticed it as well.

**Comment by nicklockwood** [comment]
@jshier @mitjacendak fixed in 0.51.4
